src/Server.ts

```diff
@@ -1,15 +1,18 @@
-import { IConfig } from './config/IConfig';
+import { IConfig, notFoundRoute, errorHandler } from './index';
 import * as express from 'express';
+import bodyParser = require ('body-parser');
 
 class Server{
-  app = express();
+  public app: express.Express;
   private port: string;
```

Comment:
remove this whenever you need port you can get from "this.config.port". So don't do redundancy